TASVideos/Extensions/ApplicationBuilderExtensions.cs

```diff
@@ -48,6 +48,31 @@ public static IApplicationBuilder UseStaticFilesWithExtensionMapping(this IAppli
 
 	public static IApplicationBuilder UseMvcWithOptions(this IApplicationBuilder app, IHostEnvironment env)
 	{
+		const string UserAgentReportURL = "https://tasvideos.org/Diagnostics/UserAgentInterventionReports";

```

Comment:
Ideally this would be `localhost` when running locally and `tasvideos.org` in prod. Is there a better way to do that than checking `env.IsProduction()`?